src/utils/api/transaction/lsk.js

```diff
@@ -332,26 +309,23 @@ export const createMultiSignatureTransaction = ({
   } = transactionObject;
 
   const schema = moduleAssetSchemas[moduleAssetId];
-  if (moduleAssetId === MODULE_ASSETS_NAME_ID_MAP.registerMultisignatureGroup) {
```

Comment:
The mandatory an optional keys need to be a part of the asset in case of the `registerMultisignatureGroup` transaction